hazelcast/src/main/java/com/hazelcast/client/impl/protocol/task/map/MapExecuteWithPredicateMessageTask.java

```diff
@@ -45,9 +44,11 @@ public MapExecuteWithPredicateMessageTask(ClientMessage clientMessage, Node node
 
     @Override
     protected OperationFactory createOperationFactory() {
+        MapOperationProvider operationProvider = getOperationProvider(parameters.name);
         EntryProcessor entryProcessor = serializationService.toObject(parameters.entryProcessor);
         Predicate predicate = serializationService.toObject(parameters.predicate);
-        return new PartitionWideEntryWithPredicateOperationFactory(parameters.name, entryProcessor, predicate);
+        return operationProvider.
```

Comment:
This is a flavor thing, I prefer to write the method name after the . and put the argument underneath:
```
return operationProvider.createPartitionWideEntryWithPredicateOperationFactory(
      parameters.name, entryProcessor, predicate);
 ```

This way it is immediately obvious you have a method call